gradle.properties

```diff
@@ -8,4 +8,4 @@ jettyRepackagedVersion=9.4.12.v20180830
 seleniumVersion=4.1.1
 mockserverNettyVersion=5.5.1
 
-labkeySchemasTestVersion=21.7.0
+labkeySchemasTestVersion=22.3-SNAPSHOT
```

Comment:
This should really be 22.3.0-SNAPSHOT, shouldn't it? 

The version from Adam's branch is actually not going to get published to this or any version. If we want to assure tests pass before merging, we'll need to publish with a special version number (manually) and use that here until the merge happens and a 22.3.0-SNAPSHOT version is available.